addons/account_peppol/static/src/components/res_config_settings_buttons/res_config_settings_buttons.xml

```diff
@@ -12,14 +12,6 @@
                             <t t-out="createUserButtonLabel"/>
                     </button>
                 </div>
-                <div class="mt-3">
-                    <button type="button"
```

Comment:
Just a small note, in theory this could break customization in case someone made an xpath on this.
In this case we consider it's extremely unlikely so we'll do it regardless of the stable policy but keep that in mind :wink: 
You can also remove the "migrate" function from JS. Since Owl component are rendered dynamically, the page only needs to be reloaded to get the updated component, you don't need to -u as for QWEB templates, therefore no need to keep the dead `migrate`.
The `button_migrate_peppol_registration` is therefore inaccessible from the standard Odoo (no custo), but we put the message as a safeguard + for customizations.